app/models/mixins/inter_region_api_method_relay.rb

```diff
@@ -80,6 +80,11 @@ def self.exec_api_call(region, collection_name, action, api_args = nil, id = nil
       result.attributes
     when ManageIQ::API::Client::Resource
       instance_for_resource(result)
+    when Hash
```

Comment:
I think that this is the wrong solution.
`Vm` retirement returns an action result, which I believe is correct.
However, retiring anything else (see api/base_controller/generic.rb) is returning the resource, which I believe is wrong.
I think retirement of anything should return the action result.  @abellotti Any thoughts on this?